contributors/index.js

```diff
@@ -16,11 +24,11 @@ module.exports.list = [
     twitter_username: "#"
   },
   {
-    name: "Wisnu Febriadi",
-    country: "Indonesia",
+    name: "Dhanam Parekh",
+    country: "India",
     role: "To be filled",
```

Comment:
hi @Dhanam77 Can you fill in the `"role"` property with what type of developer you are or want to be !